### PR TITLE
pick 1.5.2

### PR DESCRIPTION
**Pull request by teoljungberg**

### CODE REVIEW

**Comment by teoljungberg** [comment]
Regarding the build failure - `make check` passed for me locally on El Capitan. Is there anything I can do and/or help with to get the build back to green?

File: Formula/pick.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Pick < Formula
   desc "Utility to choose one option from a set of choices"
-  homepage "https://github.com/thoughtbot/pick"
-  url "https://github.com/thoughtbot/pick/releases/download/v1.4.0/pick-1.4.0.tar.gz"
-  sha256 "46f46b0df54cf27e8dd19ae291d5534cb55ef37d9cdb3cc774cd88c809f718fd"
+  homepage "https://github.com/calleerlandsson/pick"
+  url "https://github.com/calleerlandsson/pick/releases/download/v1.5.1/pick-1.5.1.tar.gz"
+  version "1.5.1"
```
Comment (teoljungberg) [inline Formula/pick.rb]: Good call, done!

**Comment by MikeMcQuaid** [comment]
@BrewTestBot test this please

**Comment by MikeMcQuaid** [comment]
> Regarding the build failure - make check passed for me locally on El Capitan. Is there anything I can do and/or help with to get the build back to green?

@teoljungberg I'm not sure what needs to be done, I'm afraid, but it'll need to be fixed before we can 🚢 this.

**Comment by calleerlandsson** [comment]
@teoljungberg In the hopes of it fixing the CI issues, I've released [v1.5.2](https://github.com/calleerlandsson/pick/releases/tag/v1.5.2). Do you want to update this PR to reference it or should we open a new one?

**Comment by teoljungberg** [comment]
@calleerlandsson Thanks for letting me know. I updated this PR with the new version

**Comment by teoljungberg** [comment]
@MikeMcQuaid We've fixed the test failure on CI now, I've rebased and squashed the PR.

File: Formula/pick.rb [hunk 2]
```diff
@@ -16,7 +16,7 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}"
-    system "make", "check"
+    system "TERM=xterm make check"
```
Comment (MikeMcQuaid) [inline Formula/pick.rb]: Use `ENV["TERM"] = "xterm"` and leave the system as-is, thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/pick.rb (hunk 2 above)
Comment (teoljungberg) [inline Formula/pick.rb]: @MikeMcQuaid I moved it to the beginning of the `install` method.

File: Formula/pick.rb [hunk 3]
```diff
@@ -13,10 +13,11 @@ class Pick < Formula
   end
 
   def install
+    ENV["TERM"] = "xterm"
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
                           "--prefix=#{prefix}"
-    system "make", "check"
+    system "make check"
```
Comment (MikeMcQuaid) [inline Formula/pick.rb]: Please put this back to how it was before, thanks.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/pick.rb (hunk 3 above)
Comment (teoljungberg) [inline Formula/pick.rb]: Done and done.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**